### PR TITLE
docs(docs-infra): fix the stackblitz on page i18n-example

### PR DESCRIPTION
**Pull request by GuillaumeBonnet**
Pasted from the issue #45066
> Description
> The provided I18n live example does not run succesfully and raises an error.
>
>What is the affected URL?
>https://angular.io/guide/i18n-example

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
The provided I18n live example does not run succesfully and raises an error.

Issue Number: #45066

## What is the new behavior?
The provided I18n live example runs succesfully.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/examples/i18n/angular.json [hunk 1]
```diff
@@ -113,7 +116,7 @@
             },
             // #docregion build-single-locale, build-production-french
             "fr": {
-              "browserTarget": "angular.io-example:build:fr"
+              "browserTarget": "angular.io-example:build:development:fr"
```
Comment (alan-agius4) [inline aio/content/examples/i18n/angular.json:119]:
```suggestion
              "browserTarget": "angular.io-example:build:development,fr"
```

File: aio/content/examples/i18n/angular.json [hunk 2]
```diff
@@ -82,6 +77,16 @@
...
             "development": {
+              "localize": ["en-US"],
+              "buildOptimizer": false,
+              "optimization": false,
+              "vendorChunk": true,
+              "extractLicenses": false,
+              "sourceMap": true,
+              "namedChunks": true
+            },
+            "development:fr": {
```
Comment (alan-agius4) [inline aio/content/examples/i18n/angular.json:88]: This new section is redundant as you can run the builder with multiple configurations. 

Ex: `angular.io-example:build:development,fr`

File: aio/content/examples/i18n/angular.json [hunk 3]
```diff
@@ -82,6 +77,16 @@
...
+              "localize": ["fr"],
```
Comment (alan-agius4) [inline aio/content/examples/i18n/angular.json:89]:
```suggestion
              "localize": false,
```

**Review by alan-agius4 — CHANGES_REQUESTED** [review]
Thanks for this. 

Just to minor comments.

**Review by alan-agius4 — APPROVED** [review]

**Review by alan-agius4 — APPROVED** [review]
LGTM, thanks for this.

**Comment by GuillaumeBonnet** [comment]
Thank you for the changes,

I think having the french translation showing by default is better for the i18n chapter and that's what the failing e2e test expected.

Can we add the aoi:preview label to check the result ?

### package.json
The scripts in the package.json of the example are:
```json
"start": "ng serve",
"start:fr": "ng serve --configuration=fr",
"build": "ng build",
"build:fr": "ng build --configuration=production-fr",
```

- "production-fr" is not a configuration anymore.
- does it make sense to have "build:fr" since "build" already build the two languages?
- "start:fr" doesn't make sense to have because "start" is already fr
- do we need a 'start:enUS' ?

**Comment by alan-agius4** [comment]
- `build:fr` can indeed be removed now. 
- `start:enUS` - not needed since `start` will load the default locale which is `en`.
- "start:fr" this is needed as start is `en`.

**Comment by GuillaumeBonnet** [comment]
I misunderstood the config, `start` does load in `en`.
I removed that one `build:fr` line

**Comment by GuillaumeBonnet** [comment]
test `//src/material/select:unit_tests_firefox` failed for commit [28faf51](https://github.com/angular/angular/commit/28faf5134832d5c293b00b4e296bef708f5509d1) when it passed for the previous commit [9e6cb3c](https://github.com/GuillaumeBonnet/angular/commit/9e6cb3c24dbb1bd53770406ce0906ed4e0a4d57c).
The differences are so small that I don't get why it would fail, can we restart the failing ci step ?

**Comment by alan-agius4** [comment]
> test `//src/material/select:unit_tests_firefox` failed for commit [28faf51](https://github.com/angular/angular/commit/28faf5134832d5c293b00b4e296bef708f5509d1) when it passed for the previous commit [9e6cb3c](https://github.com/GuillaumeBonnet/angular/commit/9e6cb3c24dbb1bd53770406ce0906ed4e0a4d57c).
> The differences are so small that I don't get why it would fail, can we restart the failing ci step ?

That should be a flake. I’ll restart the CI step.

**Review by alan-agius4 — APPROVED** [review]

**Review by josephperrott — APPROVED** [review]
LGTM

**Review by Hossein13M — APPROVED** [review]
everything seems OK

**Review by AndrewKushnir — APPROVED** [review]

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit cbbf13f1004f2f40bd08381d2420375da9e8c029.

**Pull request closed without merging**